cpp/include/core/MctsConfigurationBase.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+namespace core {
+
+struct  MctsConfigurationBase {
```

Comment:
Remove extra whitespace here.

This class deserves an explanatory comment, similar to the comment that is in `ConstantsBase.hpp`.